pkg/apiclient/application/forwarder_overwrite.go

```diff
@@ -17,6 +18,83 @@ import (
 	"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
 )
 
+var appFields = map[string]func(app *v1alpha1.Application) interface{}{
```

Comment:
Since this is a type that needs to be periodically maintained, for future reference, it would be good to explain what this is for, how its used by UI, and how the fields were chosen and when they need to be updated.

Reply:
Good point. Added comment here and in UI code as well